kube_obj_parser.py

```diff
@@ -404,22 +407,27 @@ def parse(self, url, endpoint):
                         continue
 
     def _kube_get(self, url, endpoint):
+        headers = {}
+        k8s_cert_existed = False
+        if os.path.exists(K8S_BEARER_TOKEN_FILE_NAME) and os.stat(K8S_BEARER_TOKEN_FILE_NAME).st_size > 0:
```

Comment:
For sake of simplicity, wouldn't it be a little bit safer to have `open` within a try/except so that any issue (eg. permissions) would get caught?

Reply:
Thanks for the comments. I have put the `open` into a try/except.